rboot-sampleproject/main.c

```diff
@@ -18,9 +18,6 @@
 
 static os_timer_t network_timer;
 
-void ICACHE_FLASH_ATTR user_rf_pre_init() {

```

Comment:
is this not needed anymore, I haven;t tested with 1.2 yet, but last couple of versions of the sdk you couldn't compile without providing this call back